inc/compatibility/elementor.php

```diff
@@ -297,93 +326,169 @@ private function get_global_color_prefix() {
 	}
 
 	/**
-	 * Is the current page has an elementor template
+	 * Returns Condition_Manager instance of the Elementor Pro.
 	 *
-	 * @param  string $location that location of the template such as single, archive etc.
-	 * @param  string $cond Template showing condition it can be product_archive, product etc.
-	 * @return bool
+	 * @return false|\ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager
 	 */
-	public static function is_elementor_template( $location, $cond ) {
-		if ( ! did_action( 'elementor_pro/init' ) ) {
-			return false;
-		}
-		if ( ! class_exists( '\ElementorPro\Plugin', false ) ) {
-			return false;
-		}
-		$conditions_manager = \ElementorPro\Plugin::instance()->modules_manager->get_modules( 'theme-builder' )->get_conditions_manager();
+	private static function get_condition_manager() {
+		if ( self::$elementor_conditions_manager === false ) {
+			if ( ! method_exists( '\ElementorPro\Modules\ThemeBuilder\Module', 'instance' ) ) {
```

Comment:
Same for this one

Reply:
I think that's duplicate with the previous one (early return for `get_condition_manager` method)?